upup/models/cloudup/resources/addons/networking.flannel/v0.7.0.yaml

```diff
@@ -0,0 +1,89 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: flannel
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    k8s-app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
```

Comment:
From my understanding, yes. Flannel is a meta-plugin that creates a bridge on the host to do the actual heavy lifting.